src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -292,7 +292,9 @@ public IEnumerable<string> EnumeratePSFiles()
                 return Enumerable.Empty<string>();
             }
 
-            return this.RecursivelyEnumerateFiles(WorkspacePath);
+            var foundFiles = new List<string>();
+            this.RecursivelyEnumerateFiles(WorkspacePath, ref foundFiles);
```

Comment:
`ref` isn't needed here, but I used it as a guide to the reader that this is effectively the result of the method.